src/vaadin-number-field.html

```diff
@@ -201,6 +212,17 @@
           }
           return super.checkValidity();
         }
+
+        _onInput(e) {
```

Comment:
Can't type negative number because the input is canceled when I type "-"

Reply:
Fixed.